<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_for_patient</class>
 <widget class="QDialog" name="Dialog_for_patient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="find">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Теперь Вы можете найти свою карточку</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>История ваших болезней</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
    </font>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>277</width>
     <height>105</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Myanmar Text</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите ваше имя: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="Name">
      <property name="font">
       <font>
        <family>Myanmar Text</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Myanmar Text</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите вашу фамилию:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Surname">
      <property name="font">
       <font>
        <family>Myanmar Text</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Номер карточки:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="idcard"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>ИЛИ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>41</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ваша карточка :</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ваше имя :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ваша Фамилия :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ваш возраст :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="Name_1">
        <property name="font">
         <font>
          <family>Myanmar Text</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="Age_1">
        <property name="font">
         <font>
          <family>Myanmar Text</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="Surname_1">
        <property name="font">
         <font>
          <family>Myanmar Text</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
